features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1133,7 +1133,10 @@
   ],
   "console.applications.scopes.update": ["internal_application_mgt_update"],
   "console.applications.scopes.delete": ["internal_application_mgt_delete"],
-  "console.applications.disabled_features": ["applications.loginFlow.legacyEditor"],
+  "console.applications.disabled_features": [
+    "applications.loginFlow.legacyEditor",
```

Comment:
Shall we add `loginFlow.ai`[1] as well?

[1] https://github.com/wso2/identity-apps/pull/6094/files#diff-e66c115a503827bbd4e1bd6ad04d6ff16ed77f158ae3608fae0f786905382f63L346-R346

Reply:
Shall i add it in the same PR?

cc: / @DonOmalVindula

Reply:
Added `"branding.ai"` as well.

cc:/ @chamathns

Reply:
+1